### PR TITLE
Add RadioButton components

### PR DESCRIPTION
**Pull request by nkuoch**
### Fixed Issues
Needed by https://github.com/Expensify/App/pull/7813#discussion_r819489038

### Tests
Tested in https://github.com/Expensify/App/pull/7813#discussion_r819489038

### PR Review Checklist
None

#### Web
image

### CODE REVIEW

File: src/components/RadioButtons.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+        super(props);
+
+        this.state = {
+            checkedValue: '',
+        };
+    }
+
+    render() {
+        return (
+            <view>
```
Comment (marcaaron) [inline src/components/RadioButtons.js:29]: Missing import for this and also incorrect case. Surprise this did not error 🤔

**Review by marcaaron — CHANGES_REQUESTED** [review]
Nice, but something looks off about `RadioButtons`

**Comment by shawnborton** [comment]
Looks pretty good, but maybe we want just a little bit less space between the radio button and the text? I think we want something like 12px between them.

**Comment by nkuoch** [comment]
Ok, changing the margin for checkbox labels too then.

Before:
image

After:
image

**Comment by nkuoch** [comment]
Updated!

File: src/styles/styles.js [hunk 2]
```diff
@@ -1982,6 +1982,17 @@ const styles = {
         backgroundColor: colors.white,
     },
 
+    radioButtonContainer: {
+        backgroundColor: themeColors.componentBG,
+        borderRadius: 10,
+        height: 20,
+        width: 20,
+        borderColor: themeColors.border,
```
Comment (shawnborton) [inline src/styles/styles.js:1990]: We've gotten feedback before that the border color for these feels way too light. Maybe that's a separate problem, but I could see bumping up the gray one step to using themeColors.icon which would make it a bit darker. Could I see a before/after on that so we can take a look? Thanks!

Comment (nkuoch) [reply src/styles/styles.js]: Before:

image


After:

image

**Comment by nkuoch** [comment]
Updated!

**Review by shawnborton — APPROVED** [review]

**Review by marcaaron — APPROVED** [review]

**Comment by mananjadhav** [comment]
I am tagging this PR to highlight an issue fixed [here](https://github.com/Expensify/App/issues/16683). All conditions in ternary expressions or left-hand operands on conditional renders, should be boolean. This PR is one of the PRs that uses conditional render with string operands, hence I am tagging it here for the contributors to check. 

We've also updated the item in the checklist with this [PR](https://github.com/Expensify/App/pull/18069) to avoid this issue in the future.

**Pull request merged**